NitroxClient/GameLogic/ExosuitModuleEvent.cs

```diff
@@ -113,20 +111,46 @@ public static void UseGrappling(ExosuitGrapplingArm grapplingArm, ExosuitArmActi
                 hook.SetFlying(true);
                 Exosuit componentInParent = grapplingArm.GetComponentInParent<Exosuit>();
 
-                if (!opHitVector.HasValue)
-                {
-                    Log.Error("No vector given that contains the hook direction");
-                    return;
-                }
-
-                hook.rb.velocity = opHitVector.Value;
                 Utils.PlayFMODAsset(grapplingArm.shootSound, grapplingArm.front, 15f);
                 grapplingArm.grapplingStartPos = componentInParent.transform.position;
                 break;
             }
             default:
-                Log.Error($"Grappling arm got an arm action he should not get: {armAction}");
+                Log.Error($"Grappling arm got an arm action it should not get: {armAction}");
                 break;
         }
     }
+
+    public static void UsePropulsion(ExosuitPropulsionArm propulsionArm, ExosuitArmAction armAction)
+    {
+        switch (armAction)
+        {
+            case ExosuitArmAction.START_USE_TOOL:
+                propulsionArm.propulsionCannon.animator.SetBool("use_tool", true);
```

Comment:
```suggestion
                propulsionArm.propulsionCannon.animator.SetBool(useToolAnimation, true);
```
Can use this for all following cases